### PR TITLE
Delete LIT6463Letter.xml

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**
LIT4867Theodore should stay

### CODE REVIEW

File: 4001-5000/LIT4867Theodore.xml [hunk 1]
```diff
@@ -63,18 +66,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <listRelation>
                <relation name="ecrm:P129_is_about" active="LIT4867Theodore" passive="PRS9475Theodoru"/>
                <relation name="saws:isAttributedToAuthor" active="LIT4867Theodore" passive="PRS5720JohnChr"/>
+               <relation name="ecrm:CLP46i_may_form_part_of" active="LIT6463Letter" passive="LIT1151Asceti"/>
             </listRelation>
```
Comment (DariaElagina) [inline 4001-5000/LIT4867Theodore.xml:70]: Please, the relation "formerlyListeAs" as in the GL

File: 4001-5000/LIT4867Theodore.xml [hunk 2]
```diff
@@ -63,18 +66,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
             <listBibl type="secondary">
                <bibl>
                   <ptr target="bm:Lusini1988patristica"/>
                   <citedRange>481-482</citedRange>
                </bibl>
                <bibl>
-                  <ptr target="bm:Bausi2007John"/>
+                  <ptr target="bm:Witakowski2007JohnChry"/>
```
Comment (DariaElagina) [inline 4001-5000/LIT4867Theodore.xml:77]: is the old bibliographical reference wrong?

**Review by DariaElagina — CHANGES_REQUESTED** [review]

File: 4001-5000/LIT4867Theodore.xml (hunk 1 above)
Comment (MarcinKrawczuk) [reply 4001-5000/LIT4867Theodore.xml:70]: Done

File: 4001-5000/LIT4867Theodore.xml (hunk 2 above)
Comment (MarcinKrawczuk) [reply 4001-5000/LIT4867Theodore.xml:77]: Yes, checked in EAe, the old one referred to John the Baptist, not Chrysostom.

**Pull request merged**